org.eclipse.xtext.dev-bom/build.gradle

```diff
@@ -9,9 +9,9 @@ dependencies {
         api "args4j:args4j:2.33"
         api "com.google.code.findbugs:jsr305:3.0.2"
         api "com.google.errorprone:error_prone_annotations:2.2.0"
```

Comment:
i assume this one needs to be changed too

Reply:
@miklossy this is still open

Reply:
@cdietrich I have just updated the version of `com.google.errorprone:error_prone_annotations` to `2.3.4`, according to
![screenshot](https://user-images.githubusercontent.com/20393472/106287290-eb4f2f00-6246-11eb-82e2-f39994e3e13a.png)